st2api/st2api/controllers/v1/aliasexecution.py

```diff
@@ -88,7 +91,19 @@ def post(self, payload):
                                              notify=notify,
                                              context=context)
 
-        return str(execution.id)
+        result = {
```

Comment:
Since we do not prevent an old hubot-stackstrom from connection to a new stackstorm I wonder what the error will even look like. If you can KB/document this i.e. `If you see X it is time to upgrade hubot-stackstorm` then it would help us.

Ideally, within some reasonable window old client should be supported by new server. However, this is not a concern for this endpoint alone.